apps/backend/src/reporting/reporting.service.ts

```diff
@@ -399,18 +399,39 @@ export class ReportingService {
 
     const cashDepositDateRange: DateRange = {
       from_date: cashDepositDates[1],
-      to_date: cashDepositDates[0]
+      to_date: config.period.to.toString()
```

Comment:
In the report config interface it is a date. Could change all of these dates as strings to proper Date type but will be a bigger PR. I think we used the strings instead of Dates for formatting/display in the logs but maybe it would make more sense to use Dates and format where needed?

Reply:
def an inconsistency. It should be just dates, I think when the event comes through the lambda it is casted to this type, 
But I am unsure. 

reconciliation input event has this as strings. so we need to fix it. 

pls report a small task ticket and make it consistent across all lambdas in a new PR?

Reply:
Yes, will do.